sdk/keyvault/keyvault-keys/review/keyvault-keys.api.md

```diff
@@ -44,26 +45,30 @@ export interface CreateKeyOptions extends coreHttp.OperationOptions {
 
 // @public
 export interface CreateRsaKeyOptions extends CreateKeyOptions {
-    hsm?: boolean;
+    hsm?: boolean; // Should we rename to hardwareProtected and deprecate hsm (breaking change)?
     keySize?: number;
+    publicExponent?: number;
+    // Do we need Name?
 }
 
 // @public
 export class CryptographyClient {
+    constructor(key: JsonWebKey);
     constructor(key: string | KeyVaultKey, credential: TokenCredential, pipelineOptions?: CryptographyClientOptions);
     decrypt(algorithm: EncryptionAlgorithm, ciphertext: Uint8Array, options?: DecryptOptions): Promise<DecryptResult>;
     encrypt(algorithm: EncryptionAlgorithm, plaintext: Uint8Array, options?: EncryptOptions): Promise<EncryptResult>;
     sign(algorithm: SignatureAlgorithm, digest: Uint8Array, options?: SignOptions): Promise<SignResult>;
     signData(algorithm: SignatureAlgorithm, data: Uint8Array, options?: SignOptions): Promise<SignResult>;
     unwrapKey(algorithm: KeyWrapAlgorithm, encryptedKey: Uint8Array, options?: UnwrapKeyOptions): Promise<UnwrapResult>;
-    readonly vaultUrl: string;
+    readonly vaultUrl: string; // Do we need to add keyId?
```

Comment:
Yes, since it helps customers figure out what key is associated with this `CryptographyClient` (they are key-specific by design).